tests/app/v1_basic_test.py

```diff
@@ -8,6 +8,7 @@
 from app import db
```

Comment:
All the tests are written with the assumption that you start with an empty DB for each test (which would be nice).  

Is that assumption true?  I assume you ran the tests two times in a row so probably but I wanted to confirm.

Reply:
The entire DB gets created from scratch when the tests run, thanks to the magic of `testing.postgresql`, so every `pytest` run gets its own db. What I'm not entirely positive about is whether we get a clean db for each in the same test run.